creator-node/src/diskManager.ts

```diff
@@ -297,7 +326,7 @@ export async function gatherCNodeUserDataToDelete(
   const cnodeUser = await models.CNodeUser.findOne({
     where: { walletPublicKey }
   })
-  if (!cnodeUser) throw new Error('No cnodeUser found')
+  if (!cnodeUser) return 0 // User is already wiped if no db record exists
```

Comment:
change makes sense just curious what made you change it - were you seeing some issues?

Reply:
yeah it was erroring after I did a forceWipe and tried to sync again. that probably won't happen in prod, but if the node is a new secondary with no info then a forceResync would short circuit with this error (if disk deletion is enabled)

Reply:
makes sense